ngi_reports/common/ign_sample_report.py

```diff
@@ -62,11 +60,6 @@ def __init__(self, config, LOG, working_dir, **kwargs):
         self.parse_picard_metrics()
         self.parse_deliveries()
 
-        self.create_aggregate_statistics()
-
-        self.LOG.debug('Plotting graphs')
-        self.make_plots()
```

Comment:
Gotcha, because the aggregate report `CommonReport` object is based on the `ign_sample_report.CommonReport`. My skim reading missed this too!

The approach is fine by me, maybe drop an extra comment in to make it easier for people _(me)_ to trace the hierarchy? :grinning: 

I guess an alternative place to move the function call to would be inside the `parse_template` def in the common script? Then it would still fire before the plots are needed in the report, but wouldn't be part of the class initiation any more..